package.json

```diff
@@ -33,7 +33,8 @@
     "storybook": "start-storybook",
     "test": "rm -rf ./unit_test && tsc -p tsconfig.test.json && web-test-runner",
     "es-lint": "node_modules/.bin/eslint \"src/**\"",
-    "actionlint": "node-actionlint"
+    "actionlint": "node-actionlint",
+    "chromatic": "npx chromatic --project-token=141e2a8401fe"
```

Comment:
Please remove this line, project-token is secret

Reply:
I removed that command